hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/FederationClientInterceptor.java

```diff
@@ -859,9 +869,8 @@ <R> Collection<R> invokeConcurrent(ClientMethod request, Class<R> clazz)
           results.put(subClusterId, clazz.cast(result));
         } catch (InterruptedException | ExecutionException e) {
           Throwable cause = e.getCause();
-          LOG.error("Cannot execute {} on {} : {}", request.getMethodName(),
-              subClusterId.getId(), cause.getMessage());
-          exceptions.put(subClusterId, e);
```

Comment:
I think is good to keep the map even though we just output the values.

Reply:
Thank you very much for your help in reviewing the code! I will improve this part of the code.